### PR TITLE
feature: populate re-implemented using processSFDT

### PR DESCRIPTION
**Pull request by kkc-dev**
1. this will populate a table with user input data
2. previous populate is also fine using this function

cc. @p-mag @AdamTombleson

### CODE REVIEW

File: src/sfdt/__tests__/populate.test.js [hunk 1]
```diff
@@ -60,3 +61,15 @@ describe('SFDT Parser', function() {
...
+
+describe('Populate', () => {
+	test('inject data in table', () => {
+		const sfdtWithInlines = getSFDT(tableInlines)
+		const data = {
+			"DATA::d7cd08cb-8162-42c6-b5de-166087e62b0d::field.list.weeks": "Monday"
+		}
+
+		const result = populate(data, sfdtWithInlines)
+		console.log('RESULT after populate----------', JSON.stringify(result, null, 2))
```
Comment (p-mag) [inline src/sfdt/__tests__/populate.test.js]: This test is doing nothing/

Comment (kkc-dev) [reply src/sfdt/__tests__/populate.test.js]: test utils and test updated @p-mag

File: src/__tests__/utils.js [hunk 2]
```diff
@@ -58,9 +60,23 @@ export const getInlines = () => {
 
 export const getFirstInlines = (sfdt) => getInline(sfdt, 0)
 
-export const getInline = (sfdt, position = 0, blockPosition) => {
+export const getInline = (sfdt, position = 0, blockPosition,
+	option = {
+		rowPosition: 0,
+		cellPosition: 0,
+		blockPositionInCell: 0
+}) => {
```
Comment (p-mag) [inline src/__tests__/utils.js]: Please format this.

Comment (kkc-dev) [reply src/__tests__/utils.js]: format updated @p-mag

**Pull request merged**